src/components/Account/index.js

```diff
@@ -0,0 +1,159 @@
+import React, { memo, useState } from 'react'
+import propTypes from 'prop-types'
+import ButtonContainer from '../ButtonContainer'
+import EditAccountModal from './EditAccountModal'
+import DeleteAccountModal from './DeleteAccountModal'
+import * as API from '../../api/user/user'
+import './styles.scss'
+
+const Account = props => {
+  const { nodeUser, mmuser, updateUser, updatePassword, history } = props
+  const [showModal, setShowModal] = useState(false)
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+  const [deleteError, setDeleteError] = useState(null)
+  const [field, setField] = useState('')
+
+  const openModal = fieldName => {
+    setShowModal(!showModal)
+    setField(fieldName)
+  }
+
+  const closeModal = () => {
+    setShowModal(false)
+  }
+
+  const handleDeleteUser = async () => {
+    try {
+      const data = { mmid: mmuser.id }
+      const id = localStorage.getItem('userId')
+      const token = localStorage.getItem('authToken')
+      const res = await API.deleteUser(data, id, token)
+      if (res && res.success) {
+        localStorage.removeItem('userId')
+        localStorage.removeItem('authToken')
+        history.push('/')
+      } else if (res && res.message) {
+        setDeleteError(res.message)
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e)
+    }
+  }
+
+  const updateAccountInfo = data => {
+    if (field === 'email') {
+      updateUser({ mmid: mmuser.id, email: data.email.trim().toLowerCase() })
+    } else if (field === 'phoneNumber') {
+      updateUser({ phoneNumber: data.phoneNumber })
```

Comment:
With phone number it's not that easy to check, since we decided that phone numbers don't need to be unique or verified. (Many similar numbers can be inserted to the DB without error, so we would need to go trough the entire list of user phone numbers to check). So I guess we need to trust the users not to typo those. But for email I'll add some kind of error message for that scenario.